### PR TITLE
Set up websocket server and middleware.

### PR DESCRIPTION
**Pull request by navilperez**
This change adds a websocket server and middleware handlers for adding
data to the server and polling the server.
A following PR will implement broadcasting.

### CODE REVIEW

File: blockly-rtc/server/websocket_server.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
```
Comment (samelhusseini) [inline blockly-rtc/server/websocket_server.js:1]: Add copyright header.

File: blockly-rtc/server/websocket_server.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * @fileoverview Node.js HTTP server for realtime collaboration.
+ * @author navil@google.com (Navil Perez)
+ */
+
+const socket = require('socket.io');
+const http = require('http');
+
+const sql = require('./db_utils');
+
+const PORT = 3001;
```
Comment (samelhusseini) [inline blockly-rtc/server/websocket_server.js]: rename to WS_PORT so it's clearer this is the web socket port.

File: blockly-rtc/src/websocket_handlers.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Realtime collaboration demo using Node.js and SQlite.
+ * @author navil@google.com (Navil Perez)
+ */
+
+import io from 'socket.io-client';
```
Comment (samelhusseini) [inline blockly-rtc/src/websocket_handlers.js:24]: Does ``socket.io-client`` need to be added to package.json?

Comment (samelhusseini) [reply blockly-rtc/src/websocket_handlers.js:24]: or is it included in the ``socket.io`` package?

Comment (navilperez) [reply blockly-rtc/src/websocket_handlers.js:24]: Discussed offline. It is imported in the socket.io package, but we are still including it in package.json to convey that it does need to be imported/

**Review by samelhusseini — APPROVED** [review]

File: blockly-rtc/src/index.js [hunk 4]
```diff
@@ -22,7 +22,7 @@
  */
 
 import * as Blockly from 'blockly';
-import {getEvents, writeEvents} from './api';
```
Comment (rachel-fenichel) [inline blockly-rtc/src/index.js:25]: Quick question since I'm out of date: where is http_handlers referenced now?

Comment (navilperez) [reply blockly-rtc/src/index.js:25]: http_handlers is currently not reference right now. WorkspaceClient can be initiated with either http_handlers or websocket_handlers and we are using websocket_handlers as the default for now.

**Pull request merged**